gradle.properties

```diff
@@ -2,11 +2,11 @@
 org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
-# check these on https://modmuss50.me/fabric.html
-minecraft_version=1.18.1
-yarn_mappings=1.18.1+build.12
+# check these on https://fabricmc.net/develop/
+minecraft_version=22w05a
+yarn_mappings=22w05a+build.1
 loader_version=0.12.12
-fabric_version=0.44.0+1.18
+fabric_version=0.46.4+1.18.2
```

Comment:
According to <https://fabricmc.net/develop/> it should be `fabric_version=0.46.4+1.18`

Reply:
Nevermind. seems the site is bugged lol